pkg/controller/appdefinition/status.go

```diff
@@ -588,3 +588,25 @@ func endpoints(req router.Request, cfg *apiv1.Config, app *v1.AppInstance) (stri
 
 	return strings.Join(endpointStrings, ", "), nil
 }
+
+func RefreshStatus(h router.Handler) router.Handler {
+	return router.HandlerFunc(func(req router.Request, resp router.Response) error {
+		appInstance := req.Object.(*v1.AppInstance)
+
+		if appInstance.Annotations["acorn.io/refresh"] == "true" {
+			delete(appInstance.Annotations, "acorn.io/refresh")
+			for i, annotation := range appInstance.Spec.Annotations {
+				if annotation.Key == "acorn.io/refresh" {
+					appInstance.Spec.Annotations = append(appInstance.Spec.Annotations[:i], appInstance.Spec.Annotations[i+1:]...)
+				}
```

Comment:
Could this be deleted the same way as `delete(appInstance.Annotations, "acorn.io/refresh")`

Reply:
delete is only for maps I think and the appInstance.Spec.Annotations is a slice of ScopedLabels